api/BaseAudioContext.json

```diff
@@ -10,9 +10,16 @@
           "chrome_android": {
             "version_added": true
           },
-          "edge": {
-            "version_added": "≤18"
-          },
+          "edge": [
+            {
+              "version_added": "79"
+            },
+            {
+              "version_added": "12",
+              "version_removed": "79",
+              "note": "The <code>BaseAudioContext</code> interface itself is not present, but many of the methods are available on the <a href='https://developer.mozilla.org/docs/Web/API/AudioContext'><code>AudioContext</code></a> interface."
```

Comment:
@ddbeck @Elchi3 WDYT?

Reply:
I guess I we should use `partial_implementation` too?

Reply:
I don't think anyone would look for it, but I guess given `BaseAudioContext` is not observable in these versions, let's add `partial_implementation`.

Reply:
Alright, done!